POS/Integration/06_CreateStore.md

```diff
@@ -115,11 +132,11 @@ The store wizard has been started, see [Start the store wizard](#start-the-store
 > [Info] To select a printer, the printer client *APS (Actindo Print System)* must be installed and configured.   
 For detailed information, see [Configure the printer client for POS](./03_ConfigurePrinter.md).
 
-1. Select a client for the pay desk in the *Client* drop-down list.
```

Comment:
Man muss den Drucker pro Paydesk einstellen, also man muss den Paydesk erst auswählen, oder? Entweder als Schritt 1 oder als Info hinzufügen? Vgl. Configure the pay desk, Schritt 1 -> Info

Reply:
done